### DISCUSSION TITLE
Logging instances when .PreventOverlapping stops a scheduled service from running

### DISCUSSION THREAD

**Discussion by ampetford** [jamesmh/coravel#337] [Q&A] [answered]
Hi

We use coravel to manage our scheduled services and I was wondering if there's any way to catch and log instances when .PreventOverlapping prevents a service from being started?

**Comment by jamesmh** [comment, marked as answer]
Hey! 

Not at the moment - this would require some added functionality. What's the particular use-case you were looking to use this for (just helps me understand the usefulness of a proposed feature, etc.)

Thanks for asking!

**Reply by ampetford** [reply]
In some instances where services are taking too long or hanging on something then it correctly stops future services from running and causing issues with race conditions etc.

We are currently trying to investigate ways to gain visibility of the services this may be affecting so we can trace the route cause.

Thanks for responding!
